scripts/qa_crowbarsetup.sh

```diff
@@ -2047,6 +2047,7 @@ function reboot_nodes_via_ipmi
                 sleep $((50 + RANDOM % 20))
 
                 ipmitool -H $ip -U root -P $pw chassis bootdev pxe
+                sleep $((50 + RANDOM % 20))
```

Comment:
please add a comment about why this is needed - otherwise it may be optimized out with a later refactoring
AND
if it is commented, it can be dropped when the issue is resolved $somewhen